### ISSUE TITLE
window is not defined v4.1.7

### ISSUE THREAD

**Issue by DWboutin** [erikras/redux-form#626]
I just updated redux-form and i got this error. It wasn't there before the update and i use redux-form server side too.

```
var isNative = window.navigator && window.navigator.product && window.navigator.product === 'ReactNative';
               ^

ReferenceError: window is not defined
    at Object.<anonymous> (/Users/mboutin2/working-directory/dev-admin/node_modules/redux-form/lib/index.js:17:16)
    at Module._compile (module.js:399:26)
    at Module._extensions..js (module.js:406:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/mboutin2/working-directory/dev-admin/node_modules/babel-register/lib/node.js:142:7)
    at Module.load (module.js:345:32)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:355:17)
    at require (internal/module.js:13:17)
    at Object.<anonymous> (/Users/mboutin2/working-directory/dev-admin/src/reducers/index.js:9:18)
    at Module._compile (module.js:399:26)
```

**Comment by erikras** [comment]
Oops. Sorry about that. Fixed in `v4.1.8`.

**Comment by DWboutin** [comment]
No problems ;) I just check for my npm-check this morning. I reverted to 4.1.6 and i can continue well! 

Thank you for your awesome package!
